.github/workflows/ci.yml

```diff
@@ -95,29 +95,51 @@ jobs:
 
   core-test:
     needs: prep-testbed
-    runs-on: ubuntu-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
+        os: [ubuntu-latest]
+        cibw_arch: ["x86_64"]
+        cpython-version: ["cp37-*"]
         python-version: [3.7]
         test-path: ${{fromJson(needs.prep-testbed.outputs.matrix)}}
     steps:
-      - uses: actions/checkout@v2
       - name: Set up Python ${{ matrix.python-version }}
         uses: actions/setup-python@v2
         with:
           python-version: ${{ matrix.python-version }}
+      - uses: actions/checkout@v2
       - name: Prepare enviroment
         run: |
           python -m pip install --upgrade pip
-          python -m pip install wheel
+          python -m pip install cibuildwheel
+          pip install pytest pytest-html pytest-cov pytest-mock \
+          pytest-repeat pytest-custom-exit-code pytest-timeout pytest-reraise
+          pip install jina
           pip install --pre docarray
-          pip install -e ".[test]"
+      - name: Build wheel
+        uses: pypa/cibuildwheel@2.6.0
```

Comment:
Could we use the latest action `v2.10.2`?

Reply:
And it seems mac arm64 is also supported, according to https://github.com/scikit-learn/scikit-learn/blob/main/.github/workflows/wheels.yml#L100-L126

Reply:
```
Error: Unable to resolve action `pypa/cibuildwheel@2.10.2`, unable to find version `2.10.2`
```
We could implement the old one first and bump version later

Reply:
> Add mac to ci? I added it to cd.

I mean M1 apple sillon.

Reply:
Since the older one does not support M1 apple sillon arch. Thus, I would recommend to use the latest one. You can refer to the example in the README https://github.com/pypa/cibuildwheel#example-setup